.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 executors:
   xcode-11:
     macos:
-      xcode: "11.3.1"
+      xcode: "11.4"
```

Comment:
I replaced Xcode 11.3.1, but you might want to add 11.4 as a new Xcode, depending on how quickly people are adopting 11.4. There were some compiler crashes that made it a little scary.